mesheryctl/internal/cli/root/filter/list.go

```diff
@@ -92,53 +98,68 @@ mesheryctl filter list 'Test Filter' (maximum 25 filters)
 					UpdatedAt := fmt.Sprintf("%d-%d-%d %d:%d:%d", int(v.UpdatedAt.Month()), v.UpdatedAt.Day(), v.UpdatedAt.Year(), v.UpdatedAt.Hour(), v.UpdatedAt.Minute(), v.UpdatedAt.Second())
 					data = append(data, []string{FilterID, FilterName, CreatedAt, UpdatedAt})
 				}
-				utils.PrintToTableWithFooter([]string{"FILTER ID", "NAME", "CREATED", "UPDATED"}, data, []string{"Total", fmt.Sprintf("%d", response.TotalCount), "", ""})
-				return nil
+				header = []string{"FILTER ID", "NAME", "CREATED", "UPDATED"}
```

Comment:
Ideally, we want to rely on the `JSON` tags defined with the struct and move away from this hardcoded list.

@Aijeyomah Something to take on as the next action item?

Reply:
> Ideally, we want to rely on the `JSON` tags defined with the struct and move away from this hardcoded list.
> 
> 
> 
> @Aijeyomah Something to take on as the next action item?

Yeah, I can

Reply:
Great will you open an issue to track this and start working (if you want to)?